src/components/Homepage/BodySection/BodySection.tsx

```diff
@@ -1,37 +1,57 @@
 import React from 'react';
-import { Section } from '../HomepageContent';
+import cn from 'classnames';
+import { SectionProps } from '../HomepageContent';
 import { BodySectionDescription } from './BodySectionDescription';
-import { BodySectionDescriptionContainer } from './BodySectionDescriptionContainer';
-import { BodySectionCard } from './Card/Card';
+import { HeroCard } from './Card/HeroCard';
+import { CallToAction } from './CallToAction';
+import { FeatureCard } from './Card/FeatureCard';
+import { SdkCard } from './Card/SdkCard';
+
+const cardTypes = {
+  hero: HeroCard,
+  feature: FeatureCard,
+  sdk: SdkCard,
+};
+
+export const BodySection = ({ section }: { section: SectionProps }) => {
+  const cards = section.cards ?? [];
+  const cardsExist = cards.length > 0;
+  const columns = section.columns;
+  const singleColumn = columns == 1;
+
+  const gridColVariants = {
```

Comment:
These three `const` values can probably move outside of the component as they don't change at all...